securedrop/debian/config/opt/securedrop/50unattended-upgrades

```diff
@@ -66,3 +66,7 @@ Dpkg::Options {
     "--force-confdef";
     "--force-confold";
 }
+
+// Set a fixed machine-id to ensure phased updates are consistent across all instances
+// see <https://discourse.ubuntu.com/t/phased-updates-in-apt-in-21-04/20345>.
+APT::Machine-ID "1ebf5f15850c540b3142f1584bdd496d";
```

Comment:
I confirmed via <https://utcc.utoronto.ca/~cks/space/blog/linux/Ubuntu2204ServerPhasedUpdates?showcomments> that `APT::Machine-ID` if empty is derived from `/etc/machine-id` but if set doesn't override it.